src/shared/components/oncoprint/DataUtils.ts

```diff
@@ -280,7 +280,7 @@ export function fillHeatmapTrackDatum<T extends IBaseHeatmapTrackDatum, K extend
     featureKey: K,
     featureId: T[K],
     case_:Sample|Patient,
-    data?: {value: number}[]
+    data?: {value: number, truncation?: string}[]
```

Comment:
Truncation is a (possibly poorly) chosen name for values that are larger or smaller than a given value (e.g, `>8`). The truncation value on data points represent the `>` or `<` symbols. Since I realize that truncation is a confusing word when regarded in genomics context, I am currently looking for a replacement term. When a more suitable term has been agreed on I am planning to rename all the `truncation` references in the code. A possibility might be `value_prefix_symbol` might be more suitable?

Reply:
ah, okay. How about `thresholdType: ">" | "<"`. That would make me fully understand if I was reading it.

Reply:
The new name thresholdType has been applied throughout the code as well as the type restriction (>" | "<").